contract-examples/contracts/ExampleRewardDistributor.sol

```diff
@@ -0,0 +1,72 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IAllowList.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract ExampleRewardDistributor is Ownable {
```

Comment:
hmm I think that would actually equivalent to calling precompile contracts directly. I specifically put this one to show how fees can be distributed to multiple addresses. I can remove this example contract and modify tests to call precompile directly?

Reply:
It would still be different than calling the precompile directly since it would be calling the precompile from a smart contract rather than an EOA.

My fear is only that people will mis-interpret this as an example of how to handle fee rewards, and I'm not sure if we would want that here.